pkgs/applications/networking/instant-messengers/zoom-us/default.nix

```diff
@@ -101,11 +102,10 @@ in mkDerivation {
 
   qtWrapperArgs = [
     ''--prefix PATH : ${makeBinPath [ coreutils glib.dev pciutils procps qttools.dev utillinux ]}''
-    ''--prefix LD_PRELOAD : ${libv4l}/lib/libv4l/v4l2convert.so''
     # --run "cd ${placeholder "out"}/share/zoom-us"
     # ^^ unfortunately, breaks run arg into multiple array elements, due to
     # some bad array propagation. We'll do that in bash below
-  ];
+  ] ++ stdenv.lib.optional enableV4l ''--prefix LD_PRELOAD : ${libv4l}/lib/libv4l/v4l2convert.so'';
```

Comment:
Would adding it to `autoPatchElf`’s `runtimeDeps` work. `LD_PRELOAD` is unwanted precisely for the reason described.

Reply:
Though https://github.com/NixOS/nixpkgs/pull/80070 will be needed for this to work.

Reply:
Oh, it is `LD_PRELOAD`ed. Disregard this comment.